### ISSUE TITLE
Additional data from joinTable is not available in cloned entity

### ISSUE THREAD

**Issue by r34117y** [riesenia/cakephp-duplicatable#39]
In my join table elements_basic_elements I have additional column "quantity" determinig quantities of each part for given element. This is how it looks in ElementsTable:

```
$this->belongsToMany('BasicElements', [
            'joinTable' => 'elements_basic_elements',
            'through' => 'ElementsBasicElements',
        ]);
```
Related BasicElements are cloned properly, but I have no `_joinData` attached to cloned entity.

**Comment by davidyell** [comment]
See https://github.com/riesenia/cakephp-duplicatable/blob/master/src/Model/Behavior/DuplicatableBehavior.php#L178-L180

**Comment by r34117y** [comment]
@davidyell - thanks for pointing this out. In my case it's rather safe. I'd add an option to set if _joinData should be preserved with default set to false. I'll create a pull request if I find some spare time.

**Comment by segy** [comment]
Released as 3.1.3
